src/sentry/api/serializers/models/project.py

```diff
@@ -708,12 +706,16 @@ def get_attrs(
         if not self._collapse(LATEST_DEPLOYS_KEY):
             deploys_by_project = self.get_deploys_by_project(item_list)
 
+        lpq_projects = realtime_metrics.get_lpq_projects()
         for item in item_list:
             attrs[item]["latest_release"] = latest_release_versions.get(item.id)
             attrs[item]["environments"] = environments_by_project.get(item.id, [])
             attrs[item]["has_user_reports"] = item.id in projects_with_user_reports
             if not self._collapse(LATEST_DEPLOYS_KEY):
                 attrs[item]["deploys"] = deploys_by_project.get(item.id)
+            attrs[item]["symbolication_degraded"] = should_demote_symbolication(
```

Comment:
could we add a span for this? and the `get_lpq_projects` call above as well? :D

Reply:
Sure! Can you review it again?